### DISCUSSION TITLE
计算机专业学习路线 | HackWay技术学习路线

### DISCUSSION THREAD

**Discussion by giscus** [lidongyx/hackwaydoc#38] [Announcements]
# 计算机专业学习路线 | HackWay技术学习路线

对计算机科学有追求的同学（读研、想进大厂或工作之余想提升自己），非常推荐你按照本学习路线花两三年的时间去深入学习，这对你今后的发展大有裨益；不过，如果你急于找工作（只有一年以内的时间），这种“科班式”的学习方式并不适合你，建议你按照“培训班式”的方式围绕企业招聘岗位去选择前端JavaScript、Java后端、Go等方向。

https://hackway.org/docs/cs/intro

**Comment by yeb666999** [comment]
想系统学习一下  感谢站长

**Comment by zou8944** [comment]
请问微信社群在哪里？

**Reply by lidongyx** [reply]
可以看顶部工具栏，有微信联系方式的哈

**Comment by earththreebody** [comment]
机缘巧合发现这么系统的计算机教学，非常感谢站长！！！

**Comment by jupitermyself** [comment]
太屌了

**Comment by zsq541012232** [comment]
太感谢了，支持!希望自己和大家都学有所获！

**Reply by lidongyx** [reply]
希望能够给更多人带来帮助

**Comment by KINGOf605** [comment]
站长的网站是目前我找到的最详细的计算机学习网站，非常有帮助，感谢！

**Reply by lidongyx** [reply]
谢谢支持~会持续不断更新，进一步完善

**Reply by mistcoversmyeyes** [reply]
还有一个没那么细致，但是广度大，创建时间长，积累比较多的自学指南   https://csdiy.wiki/  ，也推荐一波

**Reply by lidongyx** [reply]
csdiy一直都有关注的哦，我创建hackway的时间也挺久的，只是之前一直没有保持持续更新也没有对外宣传
相比较而言，我更侧重于原汁原味还原6大计算机名校的教学要求，这样也方便大家能够更全面系统了解计算机专业到底应该学哪些课程，哪些课程才是重点，学习这些课程最好有怎样的一个先后顺序
网站基本囊括了6大CS名校的所有重要课程，因为业内公认最好的计算机课基本也都来自这6所名校，由于收纳得全面，因此比csdiy涉及的课程要多
至于广度，还是做了一些取舍，比如一些深度的数学课、电子、信息等课程，6大CS名校的教学目录里都没有要求学这些内容，因此也就不会去整理这些“超纲内容”

**Comment by leookim** [comment]
好人啊

**Comment by drifter-0** [comment]
一些课程是没有视频的，我想知道如此如何学习呢？看书以及阅读提供的其他资料吗

**Comment by mjaow** [comment]
非常棒的cs课程列表，想问下有bitcoin或者Blockchain、Cryptocurrency相关的系列课程么，偏工程的，目前看网上比较乱，有CS专业出的，也有Finance专业出的

**Comment by XJYdemons** [comment]
感谢站长
